securing_apps/topics/docker/quickstart-zip.adoc

```diff
@@ -0,0 +1,22 @@
+[[_docker_quickstart_zip]]
+
+=== Docker Registry Quickstart Zip
+
+WARNING: This installation method is meant to be an easy way to get a docker registry authenticating against a keycloak server.  It is intended for development purposes only and should never be used in a production or production-like environment.
+
+The zip file installation mechanism provides a quickstart for developers who want to understand how the keycloak server can interact with the docker registry.  In order to configure the quickstart:
+
+ 1. Begin by creating a client configuration.  At this point you won't have a docker registry - the quickstart will take care of that part.
```

Comment:
Let's instruct the user to first create a new realm to avoid issues after setting Browser Flow to custom one that only contains 'HTTP Basic Authentication'.